run_cupy_combination_test.py

```diff
@@ -45,16 +45,7 @@
         'setuptools',
         pip_require,
         'cython==0.29.13'
-    ] + conf['requires'] + [
```

Comment:
Cannot remove this for CuPy tests as CuPy does not use `pip install -e .[jenkins]`.

Reply:
Fixed in a442c9fd77874fa51bedae705411e573c6ffec86.